tests/integration_tests/capture_write_lease/run.sh

```
@@ -0,0 +1,389 @@
+	esac
+
+	# check tables are created and data is synchronized
+	for i in $(seq $DB_COUNT); do
+		check_table_exists "capture_write_lease_$i.usertable" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	done
+	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
+
+	# add more data in upstream and check again
+	for i in $(seq $DB_COUNT); do
+		db="capture_write_lease_$i"
+		go-ycsb load mysql -P $CUR/conf/workload2 -p mysql.host=${UP_TIDB_HOST} -p mysql.port=${UP_TIDB_PORT} -p mysql.user=root -p mysql.db=$db
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)